pennylane_lightning/core/simulators/lightning_kokkos/bindings/LKokkosBindingsMPI_nb.hpp

```diff
@@ -0,0 +1,250 @@
+// Copyright 2025 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file LKokkosBindingsMPI_nb.hpp
+ * Defines lightning.kokkos specific MPI operations to export to Python using
+ * Nanobind.
+ */
+
+#pragma once
+#include <complex>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <nanobind/nanobind.h>
+#include <nanobind/ndarray.h>
+#include <nanobind/stl/complex.h>
+#include <nanobind/stl/string.h>
+#include <nanobind/stl/vector.h>
+
+#include "BindingsUtils_nb.hpp"
+#include "Constant.hpp"
+#include "ConstantUtil.hpp" // lookup
+#include "GateOperation.hpp"
+#include "Kokkos_Core.hpp"
+#include "MPIManagerKokkos.hpp"
+#include "MeasurementsKokkosMPI.hpp"
+#include "ObservablesKokkos.hpp"
+#include "StateVectorKokkosMPI.hpp"
+#include "TypeList.hpp"
+#include "Util.hpp" // exp2
+
+namespace nb = nanobind;
+
+namespace Pennylane::LightningKokkos::NanoBindings {
+
+/**
+ * @brief Define StateVector backends for lightning.kokkos MPI
+ */
+using StateVectorMPIBackends =
+    Pennylane::Util::TypeList<StateVectorKokkosMPI<float>,
+                              StateVectorKokkosMPI<double>, void>;
+
+/**
+ * @brief Register backend class specific bindings for MPI.
+ *
+ * @tparam StateVectorT
+ * @tparam PyClass
+ * @param pyclass Nanobind's state vector class to bind methods.
+ */
+template <class StateVectorT, class PyClass>
+void registerBackendClassSpecificBindingsMPI(PyClass &pyclass) {
+    using PrecisionT = typename StateVectorT::PrecisionT;
+    using ComplexT = typename StateVectorT::ComplexT;
+
+    using arr_c = nb::ndarray<std::complex<PrecisionT>, nb::c_contig>;
+
+    // Register gates for state vector
+    registerGatesForStateVector<StateVectorT>(pyclass);
+    registerControlledGate<StateVectorT>(pyclass);
+
+    pyclass
+        .def(
+            nb::init([](MPIManagerKokkos &mpi_manager, std::size_t num_qubits) {
+                return new StateVectorT(mpi_manager, num_qubits);
+            }))
+        .def(nb::init([](MPIManagerKokkos &mpi_manager, std::size_t num_qubits,
```

Comment:
```suggestion
            }));
        pyclass.def(nb::init([](MPIManagerKokkos &mpi_manager, std::size_t num_qubits,
```

(and similar for all other instances)